src/coreclr/System.Private.CoreLib/src/System/Attribute.CoreCLR.cs

```diff
@@ -436,6 +436,9 @@ private static AttributeUsageAttribute InternalGetAttributeUsage(Type type)
 
         private static Attribute[] CreateAttributeArrayHelper(Type elementType, int elementCount) =>
             elementType.ContainsGenericParameters ? new Attribute[elementCount] : (Attribute[])Array.CreateInstance(elementType, elementCount);
+
+        private static AmbiguousMatchException GetAmbiguousMatchException(Attribute attribute) =>
```

Comment:
Why don't using the one in `ThrowHelper`? https://github.com/dotnet/runtime/blob/2887a9c4756783b81da2cabb8a5e2402409d0495/src/libraries/System.Private.CoreLib/src/System/ThrowHelper.cs#L612

Reply:
Good call.